app/jobs/ingest_ephemera_csv_job.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class IngestEphemeraCSVJob < ApplicationJob
+  def perform(project, csvfile, basedir)
+    logger.info "Ingesting csv file #{csvfile}"
+    change_set_persister = ChangeSetPersister.new(
+      metadata_adapter: Valkyrie::MetadataAdapter.find(:indexing_persister),
+      storage_adapter: Valkyrie::StorageAdapter.find(:lae_storage)
```

Comment:
I was surprised to see `lae_storage` since I thought we'd removed that long ago. But I see it's just a copy of `disk_via_copy`. I've made a separate ticket to fix that: https://github.com/pulibrary/figgy/issues/4300

Reply:
I changed lae_storage to disk_via_copy just now, in the latest commit